saleor/graphql/menu/types.py

```diff
@@ -96,5 +96,5 @@ class MenuItemMoveInput(graphene.InputObjectType):
         description="ID of the parent menu. If empty, menu will be top level menu."
     )
     sort_order = graphene.Int(
-        description="Sorting position of the menu item (from 0 to x)."
+        description="The new relative sorting position of the item (from -inf to +inf)."
```

Comment:
Maybe we should also add an example here like e.g:  `1` moves the item one position forward, `-1` moves the item one position backward, `0` leaves the item unchanged. This could go to all `sort_order` args if we have more of them that work that way.